### ISSUE TITLE
VODs marked as highlights don't give a link

### ISSUE THREAD

**Issue by Pocq** [ItIckeYd/VodRecovery#14]
I have no idea how anything works but...
I have been using the script for a while and it does work, but I tried on other channels and VODs marked as highlights won't work, like:
https://www.twitch.tv/videos/1471567320

However, there is a script for tampermonkey: 
https://greasyfork.org/es/scripts/420212-twitch-vod-unblocker

With this I get into F12 in chrome and I'm able to find the .m3u8 link:
https://d1m7jfoe9zdc1j.cloudfront.net/2477ce91c627e468d6dd_ppomodoli_92016770020_1149177885/chunked/highlight-1471567320.m3u8

This domain already exists so I assume that because the formatting is different it doesn't find the link?

**Comment by Pocq** [comment]
Also it seems that you can upload videos on twitch like:
https://www.twitch.tv/videos/954895225

These also have a different format depending on quality settings:
1080p60
https://d2nvs31859zcd8.cloudfront.net/ppomodoli/954895225/e96b3cdb-ff54-4714-9822-bf453a1ea82b/1080p60/index-dvr.m3u8
720p60
https://d2nvs31859zcd8.cloudfront.net/ppomodoli/954895225/e96b3cdb-ff54-4714-9822-bf453a1ea82b/720p60/index-dvr.m3u8
720p30
https://d2nvs31859zcd8.cloudfront.net/ppomodoli/954895225/e96b3cdb-ff54-4714-9822-bf453a1ea82b/720p30/index-dvr.m3u8

**Comment by ItIckeYd** [comment]
@Pocq Yes, The reason the script doesn't give links for highlights is due to the m3u8 link being different (I will work on this as a feature request). I have not implemented quality yet and I don't think I will as I don't see it being necessary.. 99% of people are going to want the best quality possible so given that is already automatic I think ill just keep it like that.
